kuma/javascript/src/payments/pages/terms.jsx

```diff
@@ -0,0 +1,112 @@
+// @flow
+import * as React from 'react';
+import SubHeader from '../components/subheaders/index.jsx';
+
+export const title = 'MDN Web Docs payment terms';
+const TermsPage = () => (
+    <>
+        <SubHeader title={title} />
+        <main
+            className="contributions-page"
+            role="main"
+            data-testid="terms-page"
+        >
+            <section>
+                <h2>Payment Terms</h2>
+
+                <p>Mozilla (that’s us) operates MDN Web Docs.</p>
+                <p>
+                    These Payment Terms cover your payment to MDN.{' '}
+                    <a
+                        href="https://www.mozilla.org/en-US/about/legal/terms/mozilla/"
+                        target="_blank"
+                        rel="noopener noreferrer"
+                    >
+                        The Websites & Communications Terms of Use
+                    </a>{' '}
+                    cover the MDN service itself. If there is any inconsistency
+                    between these terms and the Websites & Communications Terms
+                    of Use, these Payment Terms apply.
+                </p>
+
+                <h2>Payment Authorization</h2>
+                <p>
+                    You can support MDN with a one-time payment, or you can
+                    choose to schedule monthly payments. Your payments are not
+                    tax deductible.
+                </p>
+                <p>
+                    In order to sign up for monthly payments, you must sign in
+                    to MDN using a GitHub account. GitHub has its own{' '}
+                    <a
+                        href="https://help.github.com/articles/github-terms-of-service/"
+                        target="_blank"
+                        rel="noopener noreferrer"
+                    >
+                        Terms
+                    </a>{' '}
+                    and{' '}
+                    <a
+                        href="https://help.github.com/articles/github-privacy-statement/"
+                        target="_blank"
+                        rel="noopener noreferrer"
+                    >
+                        Privacy Policy
+                    </a>
+                    .
+                </p>
+                <p>
+                    When you choose to support MDN, you authorize us to charge
+                    the payment card you provide for the amount you choose,
+                    either as a one-time payment, or as a recurring monthly
+                    payment.
+                </p>
+                <h2>Cancellation</h2>
+                <p>
+                    If you sign up for monthly payments, you may cancel at any
+                    time by sending an email to
+                    <a
+                        href="mailto:mdn-support@mozilla.com"
```

Comment:
See below about `contribution_support_email` so this can become...
```suggestion
                        href={`mailto:${contribution_support_email}`}
```